volumina/pixelpipeline/log.py

```diff
@@ -0,0 +1,3 @@
+import logging
+
+pixelpipeline_logger = logging.getLogger('volumina.pixelpipeline')
```

Comment:
is this better then just using `logger = logging.getLogger(__file__)` everywhere?